test_scripts/Smoke/API/012_PerfomInteraction_PositiveCase_SUCCESS.lua

```diff
@@ -400,8 +508,11 @@ runner.Step("CreateInteractionChoiceSet no VR commands with id 400", CreateInter
 runner.Title("Test")
 runner.Step("PerformInteraction with VR_ONLY interaction mode", PI_PerformViaVR_ONLY, {requestParams})
```

Comment:
@GetmanetsIrina Please update `PI_PerformViaVR_ONLY` function accordingly. Now it contains SUCCESS UI response.